.github/labeler.yml

```diff
@@ -1,57 +1,78 @@
-#
-# Add project labels to PRs
-# Invocation is done by .github/workflows/prlabeler.yml
-
-# changes in the documentation
-'Documentation':
-- changed-files:
-  - any-glob-to-any-file: 'book_source/**'
-  - any-glob-to-any-file: 'documentation/**'
-  - any-glob-to-any-file: 'CONTRIBUTING.md'
-  - any-glob-to-any-file: 'DEBUGING.md'
-  - any-glob-to-any-file: 'DEV-INTRO.md'
-  - any-glob-to-any-file: 'README.md'
-
-# Add 'Dockerfile' label to any changes in the docker directory
-'Dockerfile':
-- changed-files:
-  - any-glob-to-any-file: 'docker/**'
-
-
-# Add 'Website' label to any changes in the web directory
-'Website':
-- changed-files:
-  - any-glob-to-any-file: 'web/**'
-
-# Add 'Base' label to any changes in the base directory
-'Base':
-- changed-files:
-  - any-glob-to-any-file: 'base/**'
+# Auto-apply labels to PRs based on changed files
+# Triggered by .github/workflows/prlabeler.yml
+
+# Documentation
+'Documentation/User':
+  - changed-files:
+      - any-glob-to-any-file: 'book_source/**'
+      - any-glob-to-any-file: 'documentation/user_guides/**'
+
+'Documentation/Developer':
+  - changed-files:
+      - any-glob-to-any-file: 'CONTRIBUTING.md'
+      - any-glob-to-any-file: 'DEBUGGING.md'
+      - any-glob-to-any-file: 'DEV-INTRO.md'
+
+'Documentation/General':
+  - changed-files:
+      - any-glob-to-any-file: 'README.md'
+      - any-glob-to-any-file: 'documentation/**'
+
+# Models
+'Models/Climate':
+  - changed-files:
+      - any-glob-to-any-file: 'models/climate/**'
```

Comment:
The paths listed here need to correspond to paths that actually exist in the repo. We don't (and like likely won't in the future either) have a `models/climate/` subdirectory, so I don't see a need to define labels for it.